src/content/api/assets.mdx

```diff
@@ -0,0 +1,551 @@
+    }
+    ```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/assets/{assetId}/summary"
+>
+  ## Get Asset Summary
+
+  ### Attributes
+  No attributes.
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl https://service.centrapay.com/api/assets/L75M3L56N2PtBSt8g7uXLU/summary
+    ```
+
+    ```json
+    {
+      "id": "L75M3L56N2PtBSt8g7uXLU",
+      "issuer": "Centraperk Cafe",
+      "description": "Free Coffee",
+      "expiresAt": "2020-12-31T00:00:00.000Z",
+      "img": "https://static.centrapay.com/assets/brands/centraperk/cafe-token.png",
+      "liveness": "test"
+    }
+    ```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/accounts/{accountId}/assets"
+>
+  ## List Assets for Account
+
+  Returns a [paginated](https://docs.centrapay.com/api/pagination) list of Assets for an account. This will not return archived assets.
+
+  ### Attributes
+  <Properties>
+    <Property name="externalId" type="string">
+      The asset identifier from the issuing system.
+    </Property>
+  </Properties>
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl https://service.centrapay.com/api/accounts/Te2uDM7xhDLWGVJU3nzwnh/assets \
+      -H "X-Api-Key: $api_key"
+    ```
+
+    ```json
+    {
+      "items": [
+        {
+          "id": "L75M3L56N2PtBSt8g7uXLU",
+          "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+          "category": "giftcard",
+          "type": "epay.nzd.main",
+          "issuer": "ezipay",
+          "externalId": "23403283262",
+          "description": "$60 Acme Giftcard",
+          "productCode": "23403",
+          "initialBalance": "6000",
+          "balance": "6000",
+          "availableBalance": "6000",
+          "balanceUpdatedAt": "2021-01-01T00:00:00.000Z",
+          "expiresAt": "2020-12-31T00:00:00.000Z",
+          "createdAt": "2020-05-01T12:30:00.000Z"
+        },
+        {
+          "id": "WRhAxxWpTKb5U7pXyxQjjY",
+          "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+          "category": "money",
+          "type": "centrapay.nzd.main",
+          "liveness": "main",
+          "description": "NZD",
+          "createdAt": "2021-01-01T00:00:00.000Z",
+          "status": "active",
+          "currency": "NZD",
+          "balance": "2000"
+        },
+        {
+          "id": "Aj7rtHmd7rDeWoJgw9MPHe",
+          "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+          "type": "cca.coke.main",
+          "description": "Coke™ Token",
+          "category": "token",
+          "value": [
+            {
+              "currency": "NZD",
+              "amount": "400"
+            }
+          ],
+          "expiresAt": "2020-12-31T00:00:00.000Z",
+          "createdAt": "2020-05-01T12:30:00.000Z"
+        }
+      ]
+    }
+```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/assets/{assetId}/transactions"
+>
+  ## Listing Asset Transactions <Badge type="experimental"/>
```

Comment:
```suggestion
  ## List Asset Transactions <Badge type="experimental"/>
```

Would be more consistent - I know this is what the existing docs say though 😇

Reply:
You are right I have updated.